report/types_test.go

```diff
@@ -65,3 +65,10 @@ func TestFormatYAML(t *testing.T) {
 	testFormatYAML(t, SMS, "sms\n")
 	testFormatYAML(t, Unknown, "unknown\n")
 }
+
+func TestBadFormat(t *testing.T) {
+	buf := ([]byte)("- asdf::")
+	var f Format
+	err := yaml.Unmarshal(buf, &f)
+	assert.NotNil(t, err)
+}
```

Comment:
Why do we need this? it hasn't relevant to our codes

Reply:
I don't understand your comment.  The `Format` is a type that could have marshal/unmarshal operations. it's relevant. and this test case just tests the bad yaml format that causes the unmarshal error.

Reply:
got it